docs/docs/reference/index.md

```diff
@@ -4,12 +4,38 @@ comments: true
 
 # Reference
 
-This section of the Keptn documentation contains reference documentation for the various APIs and CRDs that are used in Keptn.
+Welcome to the Keptn documentation reference section,
+which provides comprehensive information about the Keptn Custom Resource Definitions (CRDs) and APIs used in Keptn.
 
-If you want to learn more about specific Custom Resource Definitions
-that you may need to populate to use Keptn,
-go to the
-[CRD Reference](./crd-reference/index.md) section.
+## CRD Reference
 
-If you want to look up some specific object properties or want ot know more about the internal APIs of Keptn,
-Go to the [API Reference](./api-reference/index.md) section.
+This section provides comprehensive reference information about the CRDs
```

Comment:
```suggestion
```

This sentence seems redundant since we have an identical one right above...

Reply:
Alright I will update it.